### ISSUE TITLE
grass_models_default.cfg from resourcepack doesn't effect the configuration correctly

### ISSUE THREAD

**Issue by HanPrower** [octarine-noise/BetterFoliage#156]
**MC version:** 1.12.2
**Mod version:** 2.1.10
**Forge version:** 14.23.0.2515

When adding a resourcepack with a betterfoliage/grass_models_default.cfg file it fails to update the configuration (I'm not sure if this effects every cfg or just the grass models).

For example PureBDcraft doesn't use the model block/grass instead it uses a custom model block/bdc_grass. When I update the grass_models_default.cfg in a separate resourcepack to reflect that the mod fails to read in the change. Instead you either have to delete the main config/betterfoliage.cfg file or go in to the mod options and set the Grass Block Models to "default".

Simply put added resourcepacks should effect the configuration accordingly without further "technical" fiddling from the user.
